packages/react/avatar/src/avatar.tsx

```diff
@@ -100,16 +100,12 @@ const AvatarFallback = React.forwardRef<AvatarFallbackElement, AvatarFallbackPro
     const { __scopeAvatar, delayMs, ...fallbackProps } = props;
     const context = useAvatarContext(FALLBACK_NAME, __scopeAvatar);
     const [canRender, setCanRender] = React.useState(delayMs === undefined);
-    const providedDocument = useDocument();
-    const documentWindow = providedDocument?.defaultView;
-
     React.useEffect(() => {
-      if (!documentWindow) return;
       if (delayMs !== undefined) {
-        const timerId = documentWindow.setTimeout(() => setCanRender(true), delayMs);
-        return () => documentWindow.clearTimeout(timerId);
+        const timerId = globalThis.window.setTimeout(() => setCanRender(true), delayMs);
+        return () => globalThis.window.clearTimeout(timerId);
```

Comment:
@chaance afaik browsers throttle setTimeout and setInterval  by 1000ms when browser tab is inactive or hidden. So if the timeout is set on the parent window, and user opens Document PIP from it with an avatar in it, then navigates to a different tab or app, these timeouts might get delayed or paused. that's why I've attached all timeouts to the provided document window